model_evaluation/rfc.md

```diff
@@ -0,0 +1,91 @@
+|     mode                   |   String        |   Options: [score, predict, compute_metrics]  Required: True  Description: Model Evaluation mode ->         1. score -> Do prediction and compute metrics for predictions        2. predict -> Compute predictions only over given dataset        3. compute_metrics -> Compute metrics over predictions provided by user                                                                                                                                                                         |
+|     task                   |   String        |   Options: [classification, regression, forecasting]  Required: True  Description: Task type for which the model is trained                                                                                                                                                                                                                                                                                                                                                                       |
+|     model_uri              |   String        |   Required: False (Required only if mode is score or predict)  Description: MLFlow model uri (could be of type) ->        To Fetch models from an azureml run        - runs:/<azureml_run_id>/run-relative/path/to/model        To Fetch model from azureml model registry        - models:/<model_name>/<model_version>        - models:/<model_name>/<stage>  NOTE: model_uri has been added to accommodate current ML Designer as it doesn’t allow users to drag-and-drop Registered models    |
+|     mlflow_model           |   MLFlow Model  |   Required: False (Required only if mode is score or predict)  Description: MLFlow model (either registered or output of another job)                                                                                                                                                                                                                                                                                                                                                             |
+|     test_data              |   MLTable       |   Required: True  Description: Test Dataset for model evaluation. Allowed formats: Delimited, JSON Lines, Parquet and Plain Text (for some tasks).                                                                                                                                                                                                                        |
+|     evaluation_config      |   MLTable       |   Required: False  Description: Additional config files required by model/metrics package. This data asset should contain a JSON Config file and any number artifacts required to load/run the model. The MLTable file should only mention JSON file name in path.        Contents of JSON File:        - See Additional Parameters list below                               |
+
+#### Additional Parameters :
```

Comment:
Are those the parameters that need to be in the `evaluation_config` input?

I mentioned before how I think having a single generic component versus multiple specific components should be driven by UX research https://github.com/Azure/azureml-assets/pull/168#discussion_r968888682

The issue is that we'll rely on users to browse the docs to identify the relevant parameters for their tasks.

Reply:
Agreed Jeff. While we are demo'ing this component to different stakeholders (1P and CSAs for 3P) along with UX team, we are seeking inputs on this aspect and accordingly plan to incorporate the changes post PrP and release as PrP refresh.